validation/src/main/java/se/fortnox/reactivewizard/validation/ValidationModule.java

```diff
@@ -0,0 +1,59 @@
+package se.fortnox.reactivewizard.validation;
+
+import com.google.inject.Binder;
+import com.google.inject.Scopes;
+import se.fortnox.reactivewizard.binding.AutoBindModule;
+import se.fortnox.reactivewizard.binding.scanners.InjectAnnotatedScanner;
+import se.fortnox.reactivewizard.config.ConfigFactory;
+import se.fortnox.reactivewizard.jaxrs.JaxRsMeta;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import java.util.Optional;
+
+public class ValidationModule implements AutoBindModule {
+
+    private final InjectAnnotatedScanner injectAnnotatedScanner;
+
+    @Inject
+    public ValidationModule(InjectAnnotatedScanner injectAnnotatedScanner) {
+        this.injectAnnotatedScanner = injectAnnotatedScanner;
+    }
+
+    @Override
+    public void configure(Binder binder) {
+        binder.bind(Validator.class).toProvider(() -> {
```

Comment:
Maybe remove this binding and also have guice always use the default constructor in ValidatorUtil instead of the explicit @Inject constructor. Now we have the same validator configuration in two places.

Reply:
Wouldn't that make it harder for others to override the configuration of the Validator? They would need to also change the binding of ValidatorUtil to use a provider which uses the other constructor?

Reply:
Agree, lets keep it this way.